pdb.py

```diff
@@ -289,10 +289,15 @@ def forget(self):
 
     def complete(self, text, state):
         if state == 0:
+            if GLOBAL_PDB:
+                GLOBAL_PDB._pdbpp_completing = True
```

Comment:
Is `GLOBAL_PDB` the right way to store this?
Should it get stored on pdbpp's `pdb` module instead?